boards/stm32f7508-dk/doc.txt

```diff
@@ -0,0 +1,60 @@
+/**
+ * @defgroup    boards_stm32f750n8-dk STM32F750N8-DK
+ * @ingroup     boards
+ * @brief       Support for the STM32F750N8-DK board.
+
+## Overview
+
+The ST [STM32F750N8-DK](https://www.st.com/en/evaluation-tools/stm32f7508-dk.html)
+is an evaluation board supporting a ARM Cortex-M7 STM32750N8 microcontroller
+with 340KB of RAM and 64KB of ROM Flash. This board is just a clone of the
+STM32F746G-DISCO board with a different microcontroller. The pinout
+is identical.
+
+As a result, the main features available on this board are shared with the ones
+provided by the STM32F746G-DISCO:
+- a 4"3 RGB 480×272 color LCD-TFT with capacitive touch screen,
+- an ethernet port
+- 2 user USB ports (FS and HS)
+- 2 user digital microphones
+- 128M-bit external Quas-SPI flash
```

Comment:
Can we wire that up as a MTD device in SPI mode?

Reply:
Probably but I haven't tried (yet)